device/arista/x86_64-arista_7060x6_64pe/Arista-7060X6-64PE-256x200G/th5-a7060x6-64pe.config.bcm

```diff
@@ -2622,7 +2622,7 @@ device:
         TM_PORT_UC_Q_TO_SERVICE_POOL:
             ?
                 PORT_ID: [[1,8], [11,18], [21,29], [33,40], [43,51], [55,62], [65,73], [76,84], [87,95], [99,106], [109,117], [121,128], [131,139], [143,150], [153,161], [165,172], [175,183], [187,194], [197,205], [209,216], [219,227], [231,238], [241,249], [253,260], [263,271], [274,282], [285,293], [297,304], [307,315], [319,326], [329,337], [341,348], 351]
-                TM_UC_Q_ID: [0,1,2,5,6,8,9]
+                TM_UC_Q_ID: [0,1,2,5,6]
```

Comment:
when searching the BCM file, I am still seeing some UC Q settings on queue 8 and 9. they are removed in the first commit, but somehow removed in the second one:

E.g.:

```
                PORT_ID: [[1,8], [11,18], [22,29], [33,40], [44,51], [55,62], [66,73], [77,84], [88,95], [99,106], [110,117], [121,128], [132,139], [143,150], [154,161], [165,172]]
                TM_UC_Q_ID: [8,9]
            :
                MIN_GUARANTEE_CELLS: 0
                SHARED_LIMITS: 1
                DYNAMIC_SHARED_LIMITS: 1
                SHARED_LIMIT_CELLS_STATIC: 0
                SHARED_LIMIT_DYNAMIC: ALPHA_1
                DYNAMIC_GROUP: MID_PRI_GROUP
                RESUME_OFFSET_CELLS: 2
                COLOR_SPECIFIC_LIMITS: 0
                COLOR_SPECIFIC_DYNAMIC_LIMITS: 0
                YELLOW_LIMIT_CELLS_STATIC: 0
                YELLOW_LIMIT_DYNAMIC: PERCENTAGE_750
                RED_LIMIT_CELLS_STATIC: 0
                RED_LIMIT_DYNAMIC: PERCENTAGE_625
            ?
```

Reply:
Good to remove this block as well - https://github.com/sonic-net/sonic-buildimage/blob/61081e89a9ec80095222452574eaacf6498c2270/device/arista/x86_64-arista_7060x6_64pe/Arista-7060X6-64PE-256x200G/th5-a7060x6-64pe.config.bcm#L2728-L2743